public/js/cinode-recruitment-public.js

```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
         formData.append("linkedInUrl", linkedInUrl.val());
         formData.append("state", 0);
         formData.append("currencyId", currencyId);
-        if (pipelineId){
+
+        if (pipelineId) {
           formData.append("pipelineId", pipelineId);
+        } else if (multiplePipeline) {
+          formData.append("pipelineId", multiplePipeline);
         }
-        
-        if (pipelineStageId){
-        formData.append("pipelineStageId", pipelineStageId);
+
+        if (pipelineStageId) {
+          formData.append("pipelineStageId", pipelineStageId);
+        } else if (multipleStageId) {
+          formData.append("pipelineStageId", multipleStageId);
         }
-        if (recruitmentManagerId){
+
+        if (recruitmentManagerId) {
```

Comment:
Guess I don't know how this works, where do all these come from? They just look undefined to me :)

Reply:
all of them are defined inside form, and its fetched from backend